src/hooks/useProducts.tsx

```diff
@@ -72,6 +80,18 @@ const initialProducts = [
         },
         volume: 1000000,
     },
+    {
+        Icon: IconWarning,
+        name: 'Error tracking',
+        type: 'error_tracking',
+        color: 'red',
```

Comment:
@corywatilo any heuristic to picking a products colour or is it just whatever is available?

Reply:
It's all by feels! We should prob go with orange though since the warning icon is more aligned with orange/yellow than red.